wdio/test-runner/typescript/forgot-password.test.ts

```diff
@@ -23,8 +23,9 @@ describe('forgot password', () => {
       await expect($('button[type="submit"]')).toBeDisplayed()
       await $('#email').setValue('person@place.biz')
       await $('button[type="submit"]').click()
-      await browser.waitUntil(() => {
-        document.body.innerHTML.includes('Internal Server Error')
+      await browser.waitUntil(async () => {
+        // "Internal Server Error" header
+        return (await $('body > h1')).isDisplayed()
```

Comment:
I think it reads more clearly for the test to not be reliant on "the initial forgot password page not having any `<h1>`):

```suggestion
        const h1 = await $('h1')
        const h1text = await h1.getText()
        return h1text.includes('Internal Server Error')
```